definitions/ext-snmpdevice/definition.yml

```diff
@@ -0,0 +1,9 @@
+domain: EXT
+type: SNMPDEVICE
+goldenTags: []
+compositeMetrics:
+  goldenMetrics:
+  - golden_metrics.yml
+configuration:
```

Comment:
One extra question: since the definitions don't add synthesis rules these entities will only appear if they are indexed directly.  Was that your intent (it's fine if you want to register the type), or were you expecting the entities to be synthesized out of telemetry data?

Reply:
just added a synthesis rule, hope this makes sense

Reply:
Since just the name of the identifier attribute is usually too generic to identify an entity we also require that you add a condition (that's why the build is failing), maybe the event name should be the condition here?

Reply:
@IreneMLC like so?

Reply:
Right now the synthesis engine only supports a single condition. Would name = fanTable be enough to match the right telemetry for this case?

Reply:
Hi Harry, the conditions have [this format](https://github.com/newrelic-experimental/entity-synthesis-definitions#conditions) in your case it could be something like this, but could you let us know if there's an account that we can impersonate to check this telemetry and be able to give you better guidance? thanks!

```
conditions:
    # The attribute’s value must match the provided value
    - attribute: name
      value: eventName
```

Reply:
@harrykimpel This entity (snmpdevice) does not include any synthesis rules, was that intentional? Is there a plan to index them in any other way?